contracts/options.cairo

```diff
@@ -185,6 +186,25 @@ func mint_option_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
                 strike_price=strike_price,
             );
         }
+
+        with_attr error_message("trade exceeds max liqpool deposit size"){
+            let (user_address) = get_caller_address();
+            let (optsize) = IOptionToken.balanceOf(contract_address=option_token_address, account=user_address);
+            assert optsize.high = 0;
+            
+            let (adjspd) = get_pool_volatility_adjustment_speed(lptoken_address=lptoken_address);
+            let adjspd_ = adjspd * 100;
+            assert_nn(adjspd_);
+
+            let max_opt_perc_math = Math64x61.fromFelt(MAX_OPTION_SIZE_PERCENT_OF_VOLADJSPD);
+            let hundred = Math64x61.fromFelt(100);
+            let ratio = Math64x61.div(max_opt_perc_math, hundred);
+
+            let max_optsize_m64x61 = Math64x61.mul(ratio, adjspd);
+
+            let max_optsize = toInt_balance(max_optsize_m64x61, option_token_address);
+            assert_le(optsize.low, max_optsize);
+        }
```

Comment:
also this same thing should be in the burn_option token... ie burn is closing of the position and the impact on the amm is the same as opening...